### ISSUE TITLE
Cannot install latest master

### ISSUE THREAD

**Issue by sagikazarmark** [ThreeDotsLabs/watermill#95]
Trying to install latest master using go modules and replace:

```
replace github.com/ThreeDotsLabs/watermill v0.4.0 => github.com/ThreeDotsLabs/watermill v0.4.1-0.20190601181058-54fc7f5042f7
```

I receive the following error:

```
go: github.com/nats-io/go-nats-streaming@v0.4.5: unknown revision v0.4.5
go: error loading module requirements
```

Looks like the repo has been archived and there is no such revision.

**Comment by roblaszczak** [comment]
They removed v0.4.5 release :/ fixed on master in #98. Also, please keep in mind that master is development branch and maybe not stable:)

Thanks for reporting the issue!

**Comment by sagikazarmark** [comment]
> Also, please keep in mind that master is development branch and maybe not stable:)

Sure thing, thanks for the fix
